src/cli/load.js

```diff
@@ -34,14 +51,28 @@ const loadCommand: Command = async (args, std) => {
   taskReporter.start("load");
   const failedPlugins = [];
   const loadPromises = [];
+  const cacheEmpty = new Map<pluginId.PluginId, boolean>();
   for (const name of pluginsToLoad) {
     const plugin = NullUtil.get(config.bundledPlugins.get(name));
     const task = `loading ${name}`;
     taskReporter.start(task);
     const dirContext = pluginDirectoryContext(baseDir, name);
-    const promise = plugin
-      .load(dirContext, taskReporter)
-      .then(() => taskReporter.finish(task))
+    cacheEmpty.set(name, isDirEmpty(dirContext.cacheDirectory()));
+    const pluginCtx: PluginLoadContext = {
+      plugin,
+      dirContext,
+      taskReporter,
+      task,
+    };
+    const promise = loadPlugin(pluginCtx)
+      .catch((e) => {
+        if (!cacheEmpty.get(name)) {
+          // clear the cache and try again
+          fs.emptyDirSync(dirContext.cacheDirectory());
+          return loadPlugin(pluginCtx);
+        }
```

Comment:
I think it'd be appropriate to log the error here, and report that we're re-trying

Reply:
+1; if this happens habitually for non-transient reasons, we’d want to
know about it rather than just silently retrying, which would waste
time/compute and hide bugs.